### ISSUE TITLE
Composer 2 compatibility

### ISSUE THREAD

**Issue by OzanKurt** [uvdesk/community-skeleton#355]

**Comment by PeopleInside** [comment]
+1 Following this.
Need support for Composer 2 and PHP 7.4
https://github.com/uvdesk/community-skeleton/issues/377

**Comment by vaishaliwebkul** [comment]
@papnoisanjeev 
We are getting issues while installing uvdesk on Composer 2 , we can't degrade our system each time to lower version as it's impacting other projects as well so it's better to provide the support for Composer 2 now.
![Screenshot_355](https://user-images.githubusercontent.com/43210406/116772761-a9649d80-aa6e-11eb-891b-344f14a18d4f.png)

**Comment by PeopleInside** [comment]
Great point @vaishaliwebkul

**Comment by papnoisanjeev** [comment]
@PeopleInside @vaishaliwebkul 

We are already working on it and will make it working for composer 2 soon into new release.

**Comment by vaishaliwebkul** [comment]
@PeopleInside  @OzanKurt 
We have added the support for composer 2 as you can see here https://github.com/uvdesk/composer-plugin , the changes will be released in upcoming release.

Below are the steps to follow if you want to add the **composer 2** compatibilities in the preinstalled project.
1. Before proceed, first take a backup of the project as well as the database.
2. Now check at terminal **composer 2** should be installed.
3. Go to project root directory and delete the **vendor** directory 
4. execute the command from the root directory of project : `composer update`

Now verify the composer version in composer.json [here](https://github.com/uvdesk/composer-plugin/blob/master/composer.json#L17)

All done!!

**Comment by PeopleInside** [comment]
Thank you!

**Comment by PeopleInside** [comment]
@vaishaliwebkul  @papnoisanjeev I followed the guide https://github.com/uvdesk/community-skeleton/issues/355#issuecomment-832002613 all seems had success.

I tried to create a web ticket and did not received any email notification.
I'm trying to reply to tickets and no email are sent.

I also tried to do the following commands:
**php bin/console d:s:u --force
php bin/console assets:install**

Some works has been done but again no email are sent so I'm now resuming the old backup and wait you.
